libs/pika/async_mpi/tests/unit/mpi_async_storage.cpp

```diff
@@ -541,7 +538,7 @@ int main(int argc, char* argv[])
         "The total storage will be num_ranks * localMB");
 
     cmdline.add_options()("in-flight-limit",
-        pika::program_options::value<std::uint64_t>()->default_value(64),
+        pika::program_options::value<std::uint64_t>()->default_value(256),
```

Comment:
To do: what's a reasonable value that's big enough not to deadlock but to still test the throttling facility?

Reply:
@biddisco the test seems to have passed at least once now with the higher value. I'll rerun it a few times, but assuming they all pass do you think 256 is a decent value to have here as the default?

Reply:
It timed out once more, but also passed once more. This needs more investigation, but I'm going to merge this in any case since this is already an improvement.